app/root.tsx

```diff
@@ -52,8 +53,8 @@ export const meta: MetaFunction = ({data}: {data: LoaderData}) => {
 export const links: LinksFunction = () => {
   return [
     {rel: 'icon', href: '/favicon.ico'},
-    {rel: 'stylesheet', href: tailwind},
     {rel: 'stylesheet', href: styles},
+    {rel: 'stylesheet', href: tailwind},
```

Comment:
This seems wrong. We should make our styles be able to override what's in tailwind right?

Reply:
Yeah, I'm indeed also not 100% sure. An alternative would be to add things like `@reach` to a `vendors.css` file, that we load very first. We need to be able to override `@reach`, which is currently loaded via `app`.

That being said, this is unlikely to give trouble, because if we would need to overrule tailwind, we could write more specific selectors.

Reply:
Ah, yeah, we should make a `vendors.css` file 👍 I'll handle that. Thanks!

Reply:
Alrighty, I've fixed this up well :)